include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -588,22 +706,33 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 		)
 		{
 			// TODO: investigate properly if its supposed be `size` or `size-1` (polyphase kinda shows need for `size`)
-			core::vectorSIMDi32 last(kernelX.getWindowSize().x,0,0,0);
+			core::vectorSIMDi32 last(kernelX.getWindowSize().x-1,0,0,0);
 			if (inImageType>=IImage::ET_2D)
-				last.y = kernelY.getWindowSize().x;
+				last.y = kernelY.getWindowSize().y-1;
 			if (inImageType>=IImage::ET_3D)
-				last.z = kernelZ.getWindowSize().x;
+				last.z = kernelZ.getWindowSize().z-1;
 			return last;
 		}
+		static inline core::vectorSIMDu32 getPhaseSupportLUTAxisOffsets(const core::vectorSIMDu32& phaseCount,
+			const CScaledImageFilterKernel<KernelX>& kernelX,
+			const CScaledImageFilterKernel<KernelY>& kernelY,
+			const CScaledImageFilterKernel<KernelZ>& kernelZ)
+		{
+			core::vectorSIMDu32 result;
+			result.x = 0u;
+			result.y = (phaseCount[0] * kernelX.getWindowSize().x) * sizeof(value_type);
+			result.z = ((phaseCount[0] * kernelX.getWindowSize().x) + (phaseCount[1] * kernelY.getWindowSize().y)) * sizeof(value_type);
```

Comment:
can we keep DRY and write it as
```cpp
core::vectorSIMDu32 result(0u);
result.y = phaseCount.x * kernelX.getWindowSize().x;
result.z = result.y + phaseCount.y * kernelY.getWindowSize().y;
return result*sizeof(value_type);
```

Reply:
also use `real_window_size` as input instead of the 3 kernels? (less opportunity for fuckups)

Reply:
Resolved in #343.